### PR TITLE
fix(tip): set significant to 6 for formatting max amount

### PR DESCRIPTION
**Pull request by UncleBill**
closes [#MF-322](https://mask.atlassian.net/browse/MF-322)

### CODE REVIEW

File: packages/mask/src/plugins/NextID/components/Tip/TokenSection/index.tsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ export const TokenSection: FC = () => {
             token={token}
             amount={amount}
             maxAmount={maxAmount}
+            maxAmountDecimals={6}
```
Comment (guanbinrui) [inline packages/mask/src/plugins/NextID/components/Tip/TokenSection/index.tsx]: Some tokens have more decimals than 6. You can read the `decimals` of original tokens here.

Comment (UncleBill) [reply packages/mask/src/plugins/NextID/components/Tip/TokenSection/index.tsx]: Of course.
In fact, I used a wrong name, it should be `maxAmountSignificant` instead.

**Review by Jack-Works — DISMISSED** [review]

**Review by guanbinrui — DISMISSED** [review]

**Comment by UncleBill** [comment]
@DimensionDev/qa Please help to verify.

File: packages/mask/src/plugins/NextID/components/Tip/TokenSection/index.tsx [hunk 2]
```diff
@@ -1,28 +1,58 @@
...
+                          maxPriorityFeePerGas: gasConfig?.maxPriorityFeePerGas || '1',
+                      }
+                    : {
+                          gas: GAS_LIMIT,
+                          gasPrice: gasPrice || defaultGasPrice,
+                      },
+            )
+        } else {
+            setGasConfig(undefined)
+        }
```
Comment (guanbinrui) [inline packages/mask/src/plugins/NextID/components/Tip/TokenSection/index.tsx]:
```suggestion
            setGasConfig(
                isEIP1559Supported(chainId)
                    ? {
                          gas: GAS_LIMIT,
                          maxFeePerGas: gasConfig?.maxFeePerGas || defaultGasPrice,
                          maxPriorityFeePerGas: gasConfig?.maxPriorityFeePerGas || '1',
                      }
                    : {
                          gas: GAS_LIMIT,
                          gasPrice: gasPrice || defaultGasPrice,
                      },
            )
        } else {
            setGasConfig(undefined)
        }
```

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**